.vib/owncloud/cypress/cypress/integration/owncloud_spec.js

```diff
@@ -4,77 +4,18 @@ import { random } from '../support/utils';
 it('allows creating a folder and uploading a file ', () => {
   cy.login();
   cy.get('.new').click();
-  cy.get('[data-action="folder"]').click();
+  cy.contains('Folder').click();
```

Comment:
Could we use a `visit` here to start the test from the "New Folder" page and avoid these 2 clicks?

Reply:
It's not possible, the app uses javascript to send the form, there is no direct URL.